package.json

```diff
@@ -24,6 +24,7 @@
   },
   "devDependencies": {
     "dotenv-cli": "^7.4.1",
+    "eslint": "^8.57.0",
```

Comment:
I don't think this is required here, as the subpackages contain `eslint` themselves and this ends up not being used. Or did you encounter some problem with the setup without this?